src/test/java/LoginTests.java

```diff
@@ -8,14 +10,20 @@
 public class LoginTests extends BaseTest {
 
     @Test
-    public static void LoginEmptyEmailPasswordTest () {
+    public void Navigation () {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example` public void navigation ()`